randovania/games/prime2/generator/base_patches_factory.py

```diff
@@ -171,3 +180,46 @@ def configurable_node_assignment(self, configuration: EchoesConfiguration, game:
             ])))
 
         return result
+
+    def assign_save_door_weaknesses(self, patches: GamePatches, configuration: EchoesConfiguration, game: GameDescription) -> GamePatches:
+        if not configuration.blue_save_doors:
+            return patches
+
+        nic = NodeIdentifier.create
+        power_weak = game.dock_weakness_database.get_by_weakness("door", "Normal Door (Forced)")
+
+        save_doors = [
+            nic("Agon Wastes", "Save Station A", "Door to Mining Plaza"),
```

Comment:
I did consider it, but they account for 4/5 save stations in Agon. It kinda defeats the point if that many save stations are excluded in an area. My rule for what gets included was just "areas that are named save station" - this excludes Dark Falls and Landing Site, but keeps the ones in agon

Reply:
I would keep the _blue_ door of these save stations still blue.

Reply:
if it *must* be one or the other, it should really be the missile door that gets changed. the missile door is very much the primary entrance for each of them and the one that will cause annoyance if inaccessible

Reply:
The multi-entrance doors are tricky because of the different paths/requirements to reach either side. I'm not really sure where I stand on this. Ideally all stations would be blue but that might alter logic too much. For now, we could exclude these and see how things play out, or change the Light World doors only? Just throwing ideas around. Alternatively, make the Missile doors be the same for Light and Dark. Unsure if that's related though.

Reply:
hm... eh, should be fine to keep all doors blue. Trickless gets a bit more places where missile launcher can be

Reply:
I really, really don't want to exclude them. I think the logic impacts are minimal, and excluding them would make *every* save station in Agon require either bombs or missiles to access in standard presets. It just defeats the point

Reply:
Totally fair. More options doesn't hurt.